curvefs/src/client/inode_wrapper.cpp

```diff
@@ -358,8 +365,8 @@ CURVEFS_ERROR InodeWrapper::UnLinkLocked(uint64_t parent) {
 
 CURVEFS_ERROR
 InodeWrapper::UpdateInodeStatus(InodeOpenStatusChange statusChange) {
-    MetaStatusCode ret =
-        metaClient_->UpdateInodeAttrWithOutNlink(inode_, statusChange);
+    MetaStatusCode ret = metaClient_->UpdateInodeAttrWithOutNlink(
```

Comment:
UpdateInodeStatus this function is not used anymore, you can delete it.

Reply:
fix